qa/tasks/ses_qa.py

```diff
@@ -142,6 +142,40 @@ def begin(self):
                 raise ConfigError(self.err_prefix + "No such method ->{}<-"
                                   .format(method_spec))
 
+    def drive_replace_initiate(self, **kwargs):
+        """
+        Initiate Deepsea drive replacement
+
+        Assumes there is 1 drive not being deployed (1node5disks - with DriveGroup `limit: 4`)
+
+        In order to "hide" an existing disk from the ceph.c_v in teuthology
+        the disk is formatted and mounted.
+        """
+        # :TODO: make this random and find correct disk device
+        osd_id = "2"
+        cmd = """sudo ceph osd tree -f json-pretty |
+                 jq '[.nodes[] | select(.type == \"osd\")] | length'"""
+        total_osds = self.master_remote.sh(cmd)
+        assert int(total_osds) == 4
+        self.master_remote.sh("sudo ceph osd tree --format json > before.json")
```

Comment:
instead of `> before.json` please use `| tee before.json` so the output of the command is shown in the log